app/pages/project/home.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import FinishedBanner from './finished-banner';
+import ProjectMetadata from './metadata';
+import ProjectHomeWorkflowButtons from './home-workflow-buttons';
+import markdownz from 'markdownz';
+const Markdown = markdownz.Markdown;
```

Comment:
I've run into this before: markdownz doesn't export Markdown and MarkdownEditor, so you ed up having to do this https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/components/wrapped-markdown.jsx#L4

I think @rogerhutchings had an idea why this was happenng, but I can't remember what it was.